benchmarks/image-processing/plugins/gatsby-source-remote-images/fetch-image-urls.js

```diff
@@ -0,0 +1,41 @@
+const { Sema } = require(`async-sema`)
+const fetch = require(`node-fetch`)
+const _ = require(`lodash`)
+const fs = require(`fs`)
+const path = require(`path`)
+
+const baseUrl = `http://www.splashbase.co/api/v1/images/`
+
+const s = new Sema(
+  10, // Allow 4 concurrent async calls
```

Comment:
Comment is incorrect
```suggestion
  10, // Allow 10 concurrent async calls
```